forms/NullableField.php

```diff
@@ -1,75 +1,98 @@
 <?php
+
 /**
- * NullableField is a field that wraps other fields when you want to allow the user to specify whether the value of
- * the field is null or not.
- * 
- * The classic case is to wrap a TextField so that the user can distinguish between an empty string and a null string.
+ * NullableField is a field that wraps other fields when you want to allow the user to specify
+ * whether the value of the field is null or not.
+ *
+ * The classic case is to wrap a TextField so that the user can distinguish between an empty string
+ * and a null string.
+ *
  * $a = new NullableField(new TextField("Field1", "Field 1", "abc"));
- * 
- * It displays the field that is wrapped followed by a checkbox that is used to specify if the value is null or not.
- * It uses the Title of the wrapped field for its title.
- * When a form is submitted the field tests the value of the "is null" checkbox and sets its value accordingly.
- * You can retrieve the value of the wrapped field from the NullableField as follows:
+ *
+ * It displays the field that is wrapped followed by a checkbox that is used to specify if the
+ * value is null or not. It uses the Title of the wrapped field for its title.
+ *
+ * When a form is submitted the field tests the value of the "is null" checkbox and sets its value
+ * accordingly. You can retrieve the value of the wrapped field from the NullableField as follows:
+ *
  * $field->Value() or $field->dataValue()
- * 
- * You can specify the label to use for the "is null" checkbox.  If you want to use I8N for this label then specify it
- * like this:
- * 
- * $field->setIsNullLabel(_T(SOME_MODULE_ISNULL_LABEL, "Is Null");
- * 
+ *
+ * You can specify the label to use for the "is null" checkbox. If you want to use i18n for this
+ * label then specify it like this:
+ *
+ * $field->setIsNullLabel(_T(SOME_MODULE_ISNULL_LABEL, "Is Null"));
+ *
  * @author Pete Bacon Darwin
+ *
  * @package forms
  * @subpackage fields-basic
  */
 class NullableField extends FormField {
 	/**
 	 * The field that holds the value of this field
+	 *
 	 * @var FormField
 	 */
 	protected $valueField;
-		
+
 	/**
 	 * The label to show next to the is null check box.
+	 *
 	 * @var string
 	 */
 	protected $isNullLabel;
-	
 
 	/**
 	 * Create a new nullable field
-	 * @param $valueField
-	 * @return NullableField
+	 *
+	 * @param FormField $valueField
+	 * @param null|string $isNullLabel
 	 */
 	public function __construct(FormField $valueField, $isNullLabel = null) {
 		$this->valueField = $valueField;
-		$this->isNullLabel = $isNullLabel;
-		if ( is_null($this->isNullLabel) ) {
-			// Set a default label if one is not provided.
+
+		if($isNullLabel) {
```

Comment:
We need to support `false` for explicitly empty labels. Can we change this to `if(isset($isNullLabel))` to keep the current behaviour?